app/views/pages/resources.html.erb

```diff
@@ -1,9 +1,100 @@
 <%- @page_title = 'Tools & Resources' -%>
 
-<div class="usa-section main-section">
+<div class="usa-section main-section tools-resources">
   <h1><%= @page_title %></h1>
+  <div class="usa-grid tools-links">
+    <ul class="usa-unstyled-list usa-width-one-half tools-list">
+      <li>
+        <%= link_to weight_estimator_path do %>
+          <%= image_tag 'icons/weight-estimator.svg' %>
+          Household Goods Weight Estimator
+        <% end %>
+      </li>
+      <li>
+        <%= link_to offices_path do %>
+          <%= image_tag 'icons/office-locator.svg' %>
+          PPPO &amp; Weight Scale Locations
+        <% end %>
+      </li>
+    </ul>
+    <ul class="usa-unstyled-list usa-width-one-half tools-list">
+      <li>
+        <%= link_to '#TODO' do %>
+          <%= image_tag 'icons/inventory-form.svg' %>
+          Self-Inventory Form
+        <% end %>
+      </li>
+      <li>
+        <%= link_to '#TODO' do %>
+          <%= image_tag 'icons/ppm-estimator.svg' %>
+          PPM Reimbursement Estimator
+        <% end %>
+      </li>
+    </ul>
+  </div>
+  <div class="resources-links">
+    <div class="usa-grid">
+      <div class="usa-width-one-third">
+        <%= image_tag 'logos/military_one_source_logo.svg' %>
+      </div>
+      <div class="usa-width-two-thirds">
+        <h4><%= link_to 'Military One Source', 'https://www.militaryonesource.mil/' %></h4>
+        <p>Military One Source has a bunch of articles about how to plan your move, find housing and employement in your new location, and how to deal with the stress  caused from uprooting your family.</p>
+        <ul class="usa-unstyled-list two-column-list">
+          <li><%= link_to 'Master Your Move Guide', 'https://www.militaryonesource.mil/pcs' %></li>
```

Comment:
There's a `usa-external_link` class in the USWDS that we could possibly use here and on other third-party site links if it makes sense in the UI.